coreneuron/io/output_spikes.cpp

```diff
@@ -288,11 +289,11 @@ void output_spikes(const char* outpath,
     }
 #if NRNMPI
     if (corenrn_param.mpi_enable && nrnmpi_initialized()) {
-        output_spikes_parallel(outpath, population_name_offset);
+        output_spikes_parallel(outpath, "out", population_name_offset);
     } else
 #endif
     {
-        output_spikes_serial(outpath);
+        output_spikes_serial(outpath, "out");
     }
```

Comment:
So ascii file will be always out.dat. Its fine just want to confirm with @jorblancoa.

Reply:
correct